packages/blade/tokens/colors.ts

```diff
@@ -0,0 +1,243 @@
+import opacities from './opacities';
```

Comment:
```suggestion
import opacity from './opacity';
```

So that we can use `opacity[9]`


Colors made sense to me because colors have a range of colors and then a scale. Opacity directly has a scale and not a range of opacities. Otherwise it's like having colors.**neutrals**.**warmGrays**[100]

Reply:
Even I was confused a lot while naming. So I was looking up patterns across. The reason why it's `opacities` is that it's a scale of opacity(a.k.a range) now if it's a collection or not doesn't matter. Similarly `fonts`, `fontSizes`, `aliases` etc.